src/api/hooks/useGetAPTPrice.ts

```diff
@@ -0,0 +1,30 @@
+const COINGECKO_API_ENDPOINT = "https://api.coingecko.com/api/v3/simple/price";
+
+export async function getAPTPrice(): Promise<number> {
+  const query = {
+    ids: "aptos", // CoinGecko ID for Aptos
+    vs_currencies: "usd", // Target currency
+  };
```

Comment:
Awesome, we should make this `getPrice` and take in any coingecko Id.

Then we can use it for anything that has a coingecko ID

Reply:
That makes a lot of sense. Changed that and set the default value of the argument to the aptos id, so as to not change existing implementations of the function.